packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -940,35 +974,82 @@ const CFDPasswordModal = observer(({ form_error, platform }: TCFDPasswordModalPr
-            <SuccessDialog
-                is_open={should_show_success}
-                toggleModal={closeModal}
-                onCancel={closeModal}
-                onSubmit={platform === CFD_PLATFORMS.MT5 && !is_selected_mt5_verified ? closeModal : closeOpenSuccess}
-                classNameMessage='cfd-password-modal__message'
-                message={getSubmitText()}
-                icon={
-                    <IconType
-                        platform={platform}
-                        type={account_type.type}
-                        show_eu_related_content={show_eu_related_content}
-                    />
-                }
-                icon_size='xlarge'
-                text_submit={success_modal_submit_label}
-                has_cancel={
-                    platform === CFD_PLATFORMS.MT5
-                        ? is_selected_mt5_verified && account_type.category === 'real'
-                        : account_type.category === 'real'
-                }
-                has_close_icon={false}
-                width={isMobile() ? '32.8rem' : 'auto'}
-                is_medium_button={isMobile()}
-            />
+            {/* TODO: Remove this once development is completed */}
+            {is_wallet_enabled && category === 'demo' && platform === CFD_PLATFORMS.MT5 ? (
+                <WalletCFDSuccessDialog
+                    header={getWalletHeader()}
+                    is_open={should_show_success}
+                    message={getSubmitText()}
+                    onSubmit={closeModal}
+                    submit_button_text={success_modal_submit_label}
+                    toggleModal={closeModal}
+                    wallet={cfd_details}
+                />
+            ) : (
+                <SuccessDialog
+                    is_open={should_show_success}
+                    toggleModal={closeModal}
+                    onCancel={closeModal}
+                    onSubmit={
+                        platform === CFD_PLATFORMS.MT5 && !is_selected_mt5_verified ? closeModal : closeOpenSuccess
+                    }
+                    classNameMessage='cfd-password-modal__message'
+                    message={getSubmitText()}
+                    icon={
+                        <IconType
+                            platform={platform}
+                            type={account_type.type}
+                            show_eu_related_content={show_eu_related_content}
+                        />
+                    }
+                    icon_size='xlarge'
+                    text_submit={success_modal_submit_label}
+                    has_cancel={
+                        platform === CFD_PLATFORMS.MT5
+                            ? is_selected_mt5_verified && account_type.category === 'real'
+                            : account_type.category === 'real'
+                    }
+                    has_close_icon={false}
+                    width={isMobile() ? '32.8rem' : 'auto'}
```

Comment:
shouldn't we be using is_mobile from store instead of isMobile ?

Reply:
@arshad-rao-deriv This is a very old component and I don't think CFD package supports useStore at the moment. Hence why I am using `isMobile()`.